notebooks/privacy_metric.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+ "cells": [
+  {
```

Comment:
What do you think about adding labels to the all the plot axes and add comments to guide the users through the notebook?